command/job_dispatch_test.go

```diff
@@ -1,6 +1,7 @@
 package command
 
 import (
+	structs2 "github.com/hashicorp/nomad/nomad/structs"
```

Comment:
What's this `structs2`? Looks like there might have been some glitch when doing the update programmatically, as I see this in a good handful of the files.

Reply:
ah woops, good catch fixed